frontend/src/modules/Traces/LatencyModalForm.tsx

```diff
@@ -44,20 +53,22 @@ const LatencyModalForm: React.FC<LatencyModalFormProps> = ({
 					<Col span={12}>
 						<Form.Item
 							name="min"
-							label="Min (in ms)"
+                            label="Min (in ms)"
+                            
 							//   rules={[{ required: true, message: 'Please input the title of collection!' }]}
 						>
-							<InputNumber />
+							<InputNumber onChange={() => setMin(parseInt(form.getFieldValue("min")))} />
 						</Form.Item>
 					</Col>
 					<Col span={12}>
 						<Form.Item name="max" label="Max (in ms)">
-							<InputNumber />
+							<InputNumber onChange={() => setMax(parseInt(form.getFieldValue("max")))} />
 						</Form.Item>
 					</Col>
 				</Row>
 				{/* </Input.Group> */}
 			</Form>
+            {disabled && <span style={{position: 'relative', top: 10, marginLeft: 200}}>max value should be greater then min value</span>}
```

Comment:
Instead of `top` maybe adding `margin` can help

Reply:
See if we can use something from antd https://ant.design/components/alert/

Reply:
`Max value should be greater than min value`

Reply:
If we go with the form item validation rules, than antd would handle the error message and we dont need to handle UI states and error rendering.

Linking the comment for better context: https://github.com/SigNoz/signoz/pull/158#discussion_r646021523